macros/edr/tests/on_run_end/merge_tests_results.sql

```diff
@@ -22,15 +22,16 @@
                                                                                    type='table') -%}
         {%- if temp_tables_union_query %}
             {{ elementary.debug_log('pulling metrics from temp metrics tables to: ' ~ database_name ~ '.' ~ schema_name ~ '.' ~ temp_metrics_table_name) }}
-            {%- do run_query(dbt.create_table_as(True, temp_table_relation, temp_tables_union_query)) %}
+            {%- do run_query(dbt.create_table_as(False, temp_table_relation, temp_tables_union_query)) %}
```

Comment:
Debug and see if we can still use temp table (understand why the table wasn't found in the next query)

Reply:
This happens because temp tables are different in redshift - "The temporary table is created in a separate, session-specific schema. (You can't specify a name for this schema.)" ([docs](https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_TABLE_NEW.html))

I saw that dbt don't use temp tables with redshift. Should we create a different flow depending on the target?